src/test/java/org/broadinstitute/consent/http/db/DarCollectionSummaryDAOTest.java

```diff
@@ -1050,4 +1056,126 @@ void testGetDarCollectionSummaryByCollectionId_ArchivedCollection() {
+
+    DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(summary.getReferenceIds().stream().findFirst().get());
+
+    createProgressReportFromDAR(dar);
+
+    List<DarCollectionSummary> summariesForDAC =
+        darCollectionSummaryDAO.getDarCollectionSummariesForDAC(chairId, dar.getDatasetIds());
+    assertTrue(summariesForDAC.get(0).getProgressReport());
+  }
+
+  @Test
+  void testGetDarCollectionSummaryForSOWithProgressReports() {
+    Pair<Pair<Integer, Integer>, DarCollectionSummary> setup = createDarCollectionSummaryForUser();
+    DarCollectionSummary summary = setup.getRight();
+
+    DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(summary.getReferenceIds().stream().findFirst().get());
+
+    createProgressReportFromDAR(dar);
+
+    User user = userDAO.findUserById(dar.getUserId());
+    Integer institutionId = user.getInstitutionId();
+
+    List<DarCollectionSummary> summariesForDAC =
+        darCollectionSummaryDAO.getDarCollectionSummariesForSO(institutionId);
+    assertTrue(summariesForDAC.get(0).getProgressReport());
+  }
+
+  @Test
+  void testGetDarCollectionSummaryForAdminWithProgressReports() {
+    Pair<Pair<Integer, Integer>, DarCollectionSummary> setup = createDarCollectionSummaryForUser();
+    DarCollectionSummary summary = setup.getRight();
+
+    DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(summary.getReferenceIds().stream().findFirst().get());
+
+    createProgressReportFromDAR(dar);
+
+    List<DarCollectionSummary> summariesForDAC =
+        darCollectionSummaryDAO.getDarCollectionSummariesForAdmin();
+    assertTrue(summariesForDAC.get(0).getProgressReport());
+  }
+
+  @Test
+  void testGetDarCollectionSummaryForDACByCollectionIdWithProgressReports() {
+    Pair<Pair<Integer, Integer>, DarCollectionSummary> setup = createDarCollectionSummaryForUser();
+    Integer chairId = setup.getLeft().getRight();
+    DarCollectionSummary summary = setup.getRight();
+
+    DataAccessRequest dar = dataAccessRequestDAO.findByReferenceId(summary.getReferenceIds().stream().findFirst().get());
+
+    createProgressReportFromDAR(dar);
+
+    DarCollectionSummary summaryForDAC =
+        darCollectionSummaryDAO.getDarCollectionSummaryForDACByCollectionId(chairId, dar.getDatasetIds(), dar.getCollectionId());
+    assertTrue(summaryForDAC.getProgressReport());
+  }
+
+  private Pair<Pair<Integer, Integer>, DarCollectionSummary> createDarCollectionSummaryForUser() {
+    Dac dac = createDac();
+    User user = createUserWithInstitution();
+    User userChair = createUserWithRoleInDac(UserRoles.CHAIRPERSON.getRoleId(), dac.getDacId());
+    Integer userId = user.getUserId();
+    Integer chairId = userChair.getUserId();
+    Integer dacId = dac.getDacId();
+    Dataset dataset = createDatasetWithDac(userId, dacId);
+
+    DarCollectionSummary summary = createDarWithVotes(userId, chairId, dataset.getDatasetId());
+    return Pair.of(Pair.of(userId, chairId), summary);
```

Comment:
This must have been fun. I don't see the need for the nested Pairs though. I might have considered a custom record class to make this a bit easier to work with:
```
public record Setup(int userId, int chairId, DarCollectionSummary summary)
```